docs/site/downloads/cldr-47.md

```diff
@@ -25,17 +25,21 @@ The most significant changes in this release are:
    - Updated time zone data to tzdata 2025a
    - [RBNF](#number-spellout-data-changes) (Number Spellout Data Improvements) for multiple languages
    - Assorted transforms improvements
-   - Updated language matching for Afrikaans to English (en) from Dutch (nl) [CLDR-18198](https://unicode-org.atlassian.net/browse/CLDR-18198)
-   - Ordered scripts in decending order of usage per locale [CLDR-18155](https://unicode-org.atlassian.net/browse/CLDR-18155)
-   - Fixed invalid codes [CLDR-18129](https://unicode-org.atlassian.net/browse/CLDR-18129)
-   - Updated population data
+   - Updated and revised population data
+   - Incorporates all changes from CLDR v46.1.
+        - [CLDR v46.1](https://cldr.unicode.org/downloads/cldr-46#461-changes) was a special release, which many users of CLDR (including ICU) have not updated to.
+So the listed changes are relative to [CLDR v46.0](https://cldr.unicode.org/downloads/cldr-46).
+       - More explicit well-formedness and validity constraints for unit of measurement identifiers
+       - Addition of derived emoji annotations that were missing: emoji with skin tones facing right
+       - Fixes to make the ja, ko, yue, zh datetimeSkeletons useful for generating the standard patterns
+       - Improved date/time test data
 
 For more details, see below.
 
 ### Locale Coverage Status
 
-CLDR 47 was a closed cycle which means that locale data changes were limited to addition of new locales, and bug fixes.
-This means that coverage levels for existing locales did not change in this release.
+CLDR 47 focused on MessageFormat 2.0 and tooling for an expansion of DDL support.
```

Comment:
I thought you wanted to move this line up to the overview section?

Reply:
Yup, changed it just now (and a couple of minor other items)